### PR TITLE
code clean up 

### PR DESCRIPTION
**Pull request by 117649**
I hope this time it is right.

### CODE REVIEW

**Comment by onemen** [comment]
It doesn't look right to me, the PR should contain only your last commit (https://github.com/onemen/TabMixPlus/pull/7/commits/6d4c93014a884c9a8c965e4304e556f6c62f9f7d) 
this is the only change you made since you last pull

**Comment by 117649** [comment]
Now nail it.

File: addon/chrome/content/preferences/prefs-ce.js [hunk 1]
```diff
@@ -1039,69 +1039,66 @@ class PrefWindow extends MozXULElement {
...
-          <button dlgtype="disclosure" class="dialog-button" hidden="true"/>`;
+          <button dlgtype="disclosure" class="dialog-button" hidden="true"/>`;          
   }
 
   connectedCallback() {
     if (this._initialized) {
       return;
     }
 
-    function updateAttribute(name, value) {
```
Comment (onemen) [inline addon/chrome/content/preferences/prefs-ce.js:1060]: Please Don't undo all my changes :)
The only bug is in updateAttribute function just replace it with this code and prefs window should work

**new function**
```js
    const updateAttribute = (name, value) => {
      if (!this.hasAttribute(name)) {
        this.setAttribute(name, value);
      }
    };
```

**diff**
```diff
-    function updateAttribute(name, value) {
-      if (this.hasAttribute(name)) {
+    const updateAttribute = (name, value) => {
+      if (!this.hasAttribute(name)) {
         this.setAttribute(name, value);
       }
-    }
+    };
```

**Review by onemen — CHANGES_REQUESTED** [review]
Thank you, just some minor changes
- Replace https://github.com/onemen/TabMixPlus/pull/7/commits/171f7052870a9b6e8fe6c81bb2e2517151116714 with commit that address the fix according to my comment.
- Move styles changes from: https://github.com/onemen/TabMixPlus/pull/7/commits/779ab829ec8515ba05f36073138c5b656d060211,  https://github.com/onemen/TabMixPlus/pull/7/commits/cc5f2b390229fc5463c1bb0ffca7924051f0af76 to separate PR

**Pull request merged**